osh.spec

```diff
@@ -234,19 +234,40 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
+%attr(755,root,root) %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%attr(755,root,root) %{python3_sitelib}/osh/hub/scripts/umb-emit.py
 %{python3_sitelib}/osh/hub
 %{_unitdir}/osh-retention.*
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+# These file should be readable and writable by respective groups.
 %dir %attr(775,root,root) %{_localstatedir}/log/osh
 %dir %attr(775,root,apache) %{_localstatedir}/log/osh/hub
-%ghost %attr(640,apache,apache) %{_localstatedir}/log/osh/hub/hub.log
 %attr(775,root,apache) %{_sharedstatedir}/osh/hub
+%ghost %attr(644,apache,apache) %{_localstatedir}/log/osh/hub/hub.log
+# These files contain secrets and should not be readable by others.
 %ghost %attr(640,root,apache) %{_sharedstatedir}/osh/hub/secret_key
 %ghost %attr(640,root,apache) %{_sysconfdir}/osh/hub/secrets/jira_secret
 %ghost %attr(640,root,apache) %{_sysconfdir}/osh/hub/secrets/bugzilla_secret
 
+%exclude %{python3_sitelib}/osh/hub/static/admin/css/vendor/select2/LICENSE-SELECT2.md
+%exclude %{python3_sitelib}/osh/hub/static/admin/fonts/LICENSE.txt
+%exclude %{python3_sitelib}/osh/hub/static/admin/img/LICENSE
+%exclude %{python3_sitelib}/osh/hub/static/admin/js/vendor/jquery/LICENSE.txt
+%exclude %{python3_sitelib}/osh/hub/static/admin/js/vendor/select2/LICENSE.md
+%exclude %{python3_sitelib}/osh/hub/static/admin/js/vendor/xregexp/LICENSE.txt
```

Comment:
Are all the `%exclude` lines really needed? I have never seen this in any other package.

Reply:
These files are listed twice in `rpmbuild` output.